paymentsheet/src/test/java/com/stripe/android/paymentsheet/PaymentSheetViewModelTest.kt

```diff
@@ -715,26 +714,26 @@ internal class PaymentSheetViewModelTest {
             .isNotNull()
     }
 
-    @Test
-    fun `fragmentConfig when all data is ready should emit value`() {
-        val viewModel = createViewModel()
-
-        viewModel._isGooglePayReady.value = true
-        viewModel._isLinkEnabled.value = true
-
-        val configs = mutableListOf<FragmentConfig>()
-        viewModel.fragmentConfigEvent.observeForever { event ->
-            val config = event.getContentIfNotHandled()
-            if (config != null) {
-                configs.add(config)
-            }
-        }
-
-        idleLooper()
-
-        assertThat(configs)
-            .hasSize(1)
-    }
+//    @Test
```

Comment:
And here

Reply:
Oh, I actually wanted to rewrite this one. Will do in the next commit!